packages/@uppy/dashboard/src/components/FileList.jsx

```diff
@@ -17,50 +18,28 @@ function chunks (list, size) {
   return chunked
 }
 
-export default (props) => {
+export default ({
+  id, error, i18n, uppy, files, acquirers, resumableUploads, hideRetryButton, hidePauseResumeButton, hideCancelButton,
```

Comment:
formatting seems a bit off here?

Reply:
Not sure which formatting? Eslint is not complainign.